stack.go

```diff
@@ -111,19 +113,27 @@ func (c Call) Format(s fmt.State, verb rune) {
 		buf := [6]byte{}
 		s.Write(strconv.AppendInt(buf[:0], int64(line), 10))
 
-	case 'n':
+	case 'n', 'k':
```

Comment:
I think the code is already complicated enough when `case 'n'` is by itself and would rather see a separate `case 'k'`. I also think this is a good idea since the `path` and `pkg` vars are not needed for `case 'n'`.